spec/apis/hackney_api/repairs_client_spec.rb

```diff
@@ -623,4 +623,62 @@
         .to be == results
     end
   end
+
+  describe '#get_keyfax_url' do
+    let(:current_page_url) { "https://repairs-hub.hackney.gov.uk/properties/00004769/repair_requests/new" }
+
+    subject { api_client.get_keyfax_url(current_page_url) }
+
+    context 'successful response' do
+      before { stub_request(:get, "#{base_url}/#{api_version}/keyfax/get_startup_url/?returnurl=#{current_page_url}").to_return(body: empty_response_body.to_json) }
+
+      it 'returns successful response body' do
+        expect(subject).to eq(empty_response_body)
```

Comment:
use explicit example instead of `empty_response_body` (like `{celia: 'thebest'}`).

Reply:
:sparkles: thebest